JPS_Ontology/ontology/ontobuildingstructure/OntoBuildingStructure.csv

```diff
@@ -0,0 +1,49 @@
+hasQuantity ,"Object Property, Asymmetric Property",,,http://www.w3.org/2002/07/owl#Thing,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,"Links any entity to a specific quantity defined in the OM ontology. This relationship is not intended to be stated explicitly, but inferred from its sub-properties. For application, only the sub-properties are used.",http://www.theworldavatar.com/kg/ontobuildingstructure/,has quantity
+hasArea,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,RoofArea,,A relationship between the roof of a building and the area it covers.,http://www.theworldavatar.com/kg/ontobuildingstructure/,has area of
+hasOrientation,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,,A relationship between a roof and its angle/position relative to the Sun.,http://www.theworldavatar.com/kg/ontobuildingstructure/,has orientation
+hasSlope ,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,,A relationship between a roof and its slope angle.,http://www.theworldavatar.com/kg/ontobuildingstructure/,has slope
+https://www.theworldavatar.com/kg/ontotimeseries/hasTimeSeries,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure, https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,,,https://www.theworldavatar.com/kg/ontotimeseries/,has time series
+consistsOf,"Object Property, Asymmetric Property",,,Stair,Landing UNION Railing UNION StairFlight UNION StairStructuralComponent,only,"A relationship that denotes the subcomposition of an assembly with their individual components. An example is that stairs consists of railing, landing, stair flights, and stair structural components.",http://www.theworldavatar.com/kg/ontobuildingstructure/,consists of
+isPartOf,"Object Property, Asymmetric Property",,,Landing UNION Railing UNION StairFlight UNION StairStructuralComponent,Stair,only,"A relationship that links a component to their assembly. An example is that railing, landing, stair flights, and stair structural components are part of stair.",http://www.theworldavatar.com/kg/ontobuildingstructure/,is part of 
+consistsOf,"Object Property, Asymmetric Property",,,Wall,Door UNION Window,only,"A relationship that denotes the subcomposition of an assembly with their individual components. An example is that stairs consists of railing, landing, stair flights, and stair structural components.",http://www.theworldavatar.com/kg/ontobuildingstructure/,consists of
+isPartOf,"Object Property, Asymmetric Property",,,Door UNION Window,Wall,only,"A relationship that links a component to their assembly. An example is that railing, landing, stair flights, and stair structural components are part of stair.",http://www.theworldavatar.com/kg/ontobuildingstructure/,is part of 
```

Comment:
As usual, I would recommend to keep either `consistsOf` or `isPartOf`, but not both (I would keep the former). Otherwise, writing queries for this will be quite awkward or has to rely on assumptions that may or may not be true.

Reply:
Removed isPartOf as per recommendation